packages/MarkdownEditor-Tests.package/MarkdownEmphasisDelimiterTest.class/instance/testMatchesOpenerNot.st

```diff
@@ -1,6 +1,7 @@
 tests - matching
 testMatchesOpenerNot
 
+	"Opener does not match the openerAndCloser because their length in sum is divisible by 3 and their individual length is not."
```

Comment:
```suggestion
	"Opener does not match openerAndCloser because their length in sum is divisible by 3 but their individual length is not."
```

Reply:
the correct grammar would be to keep the `the`. Thus I would rather keep it.

Reply:
I found it a bit inconsistent because `opener` has no article, whereas `openerAndCloser` has. As both are variables, I think that removing the article is fine (You also say "x does not equal y")

Reply:
"does not match" (without the "the") is used when somebody/-thing matches a person ("Jonas does not match Jakob") and "does not match the" is used when somebody/-thing matches something ("Jonas does not match the general public"). Here the `openerAndCloser` is a thing. Thus the `the` should be there. :D